atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/KeyValueService.java

```diff
@@ -290,6 +290,14 @@ void putUnlessExists(@QueryParam("tableRef") TableReference tableRef,
                          Map<Cell, byte[]> values) throws KeyAlreadyExistsException;
```

Comment:
For `putUnlessExists`, I believe that this is covered in the above javadoc (lines 265-269).